zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQuerySortRequest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest;
+
+import io.camunda.zeebe.gateway.protocol.rest.SortOrderEnum;
+
+public class SearchQuerySortRequest {
```

Comment:
🔧 I guess we can make this a [Java record](https://www.baeldung.com/java-record-keyword) instead of a Java plain class, right?